mam/v2/mam/message.c

```diff
@@ -810,7 +811,12 @@ retcode_t mam_msg_recv_packet(mam_msg_recv_context_t *ctx, trits_t *b,
   tryte_t checksum = -1;
 
   /*  absorb long trint ord; */
-  ERR_BIND_RETURN(pb3_unwrap_absorb_longtrint(&ctx->spongos, b, &ctx->ord), e);
+  {
```

Comment:
`ord` is already encoded as part of tag. Or you can `pb3_encode_longtrint` instead of `trits_put18`. Or add a new function `pb3_absorb_external_ntrytes` as it can be used in send.